packages/core-data/README.md

```diff
@@ -765,26 +765,28 @@ application, the page and the resolution details will be retrieved from
 the store state using `getEntityRecord()`, or resolved if missing.
 
 ```js
-import { useState } from '@wordpress/data';
 import { useDispatch } from '@wordpress/data';
+import { useCallback } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import { TextControl } from '@wordpress/components';
 import { store as noticeStore } from '@wordpress/notices';
 import { useEntityRecord } from '@wordpress/core-data';
 
 function PageRenameForm( { id } ) {
 	const page = useEntityRecord( 'postType', 'page', id );
-	const [ title, setTitle ] = useState( () => page.record.title.rendered );
 	const { createSuccessNotice, createErrorNotice } =
 		useDispatch( noticeStore );
 
+	const setTitle = useCallback( ( title ) => {
+		page.edit( { title } );
+	} );
```

Comment:
Should it include `page.edit` as a dependency?
```suggestion
	const setTitle = useCallback( ( title ) => {
		page.edit( { title } );
	}, [ page.edit ] );
```

In theory, it could change when the `id` changes.

Reply:
Oh great catch!